ansible/roles/prometheus-server/tasks/main.yml

```diff
@@ -0,0 +1,85 @@
+---
+
+- name: ensure {{ prometheus_group }} group exists
```

Comment:
It's a minor point but we're aiming for a high level of consistency across our provisioning moving forwards, so capitalise the task description here and in other tasks, for example:

```yml
- name: Ensure {{ prometheus_group }} group exists
```

Reply:
Better to have it correct. I have to say I struggled to find any common public naming conventions, but what I did find seemed to stick mainly to lower case. For example in [playbooks_best_practices](https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html#task-and-handler-organization-for-a-role).
But I'm happy either way.
Now updated to sentence case.